### PR TITLE
Modify: Spotify.Spotify version 1.2.38.720.ga4a70a0e

### PR DESCRIPTION
**Pull request by aisgbnok**
Checklist for Pull Requests
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/155308)

### CODE REVIEW

**Comment by aisgbnok** [comment]
https://github.com/microsoft/winget-pkgs/blob/910cce44e4f47ae648e41b9f4b5abf1f72aab09b/manifests/s/Spotify/Spotify/1.2.38.720.ga4a70a0e/Spotify.Spotify.installer.yaml#L15

https://github.com/microsoft/winget-pkgs/labels/Azure-Pipeline-Passed https://github.com/microsoft/winget-pkgs/labels/Validation-Completed

https://dev.azure.com/ms/winget-pkgs/_build/results?buildId=575615

**Comment by aisgbnok** [comment]
### Individual Installer Nodes

https://github.com/microsoft/winget-pkgs/blob/84484dc3b95e574e42f7d62b9d43e71335604a22/manifests/s/Spotify/Spotify/1.2.38.720.ga4a70a0e/Spotify.Spotify.installer.yaml#L15-L27

https://github.com/microsoft/winget-pkgs/labels/Validation-Installation-Error

https://dev.azure.com/ms/winget-pkgs/_build/results?buildId=575625

**Comment by aisgbnok** [comment]
Why does 910cce44e4f47ae648e41b9f4b5abf1f72aab09b [pass validation (comment)](https://github.com/microsoft/winget-pkgs/pull/155308#issuecomment-2133853437), [but fail validation (comment)](https://github.com/microsoft/winget-pkgs/pull/155308#issuecomment-2133953797)? The only different being `ElevationRequirement: elevationProhibited` in the YAML root vs individual installer nodes?

This is related to #154823, and doesn't seem to be an issue with the installer. It may be WinGet or validation?

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335146

Automated error analysis suggests -1978335146 may mean The installer cannot be run from an administrator context - does this package require `ElevationRequirement: elevationProhibited`? 

(Automated response - build 881.)

File: manifests/s/Spotify/Spotify/1.2.38.720.ga4a70a0e/Spotify.Spotify.installer.yaml [hunk 1]
```diff
@@ -7,24 +7,23 @@ InstallerType: exe
...
-ElevationRequirement: elevationProhibited
 Installers:
-- Architecture: x86
-  InstallerUrl: https://upgrade.scdn.co/upgrade/client/win32-x86/spotify_installer-1.2.38.720.ga4a70a0e-1222.exe
-  InstallerSha256: D3AE4895DEE3FCB82782BBB6CE5E620CB664C9656B3D549A81E9E0008C0E565B
-  ProductCode: Spotify
 - Architecture: x64
   InstallerUrl: https://upgrade.scdn.co/upgrade/client/win32-x86_64/spotify_installer-1.2.38.720.ga4a70a0e-1223.exe
   InstallerSha256: 08E3EBDF619C4AA97BF861A6F42097B1948EBBD12D6D1848C6C67D2B717429D5
-  ProductCode: Spotify
```
Comment (SpecterShell) [inline manifests/s/Spotify/Spotify/1.2.38.720.ga4a70a0e/Spotify.Spotify.installer.yaml:24]: Why are you removing the `ProductCode`? 🤔

**Comment by denelon** [comment]
@aisgbnok, I didn't see a bug filed anywhere related to the location of the "ElevationRequirement" in the manifest other than one related to install modes. Feel free to file a bug if you believe it's an improper behavior in the client over at winget-cli, but if you believe it's isolated to the validation environment, it should be filed here.

**Review by ItzLevvie — APPROVED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: See above;

**Pull request closed without merging**